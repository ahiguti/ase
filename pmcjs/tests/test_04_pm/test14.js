
use('Perl');
Perl.Use('Class::Struct');
var struct = Perl.Functions['Class::Struct'].struct;
struct('MyStruct', Perl.NewHash({
  s: '$',
  a: '@',
  h: '%'
}));

var obj = new Perl('MyStruct');
obj.s(100);
print(obj.s());
obj.a(15, 9);
print(obj.a(15));
obj.h('abc', 'fubar');
print(obj.h('abc'));

