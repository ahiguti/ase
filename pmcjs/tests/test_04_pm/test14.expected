100
9
fubar
